include/mitsuba/core/bitmap.h

```diff
@@ -380,6 +384,13 @@ class MTS_EXPORT_CORE Bitmap : public Object {
     void write_async(const fs::path &path, FileFormat format = FileFormat::Auto,
                      int quality = -1) const;
 
+    /**
+     * \brief Extract several color channels of a multi-channel
+     * bitmap and return them as a bitmap with the given pixel format
+     */
+    ref<Bitmap> Bitmap::extractChannels(PixelFormat fmt, const std::vector<int> &channels) const;
```

Comment:
Please use the `underscore_case` convention for the method / variable names:

```suggestion
    ref<Bitmap> Bitmap::extract_channels(PixelFormat fmt, const std::vector<int> &channels) const;
```